doc/system_requirements.md

```diff
@@ -277,6 +277,21 @@ Covers:
 
 Needs: dsn
 
+#### Uploading InputStream to a File in BucketFS
+`req~uploading-input-stream-to-a-file-in-bucketfs~1`
+
+ETC uploads the contents of an InputStream to a file inside a bucket.
+
+Rationale:
+
+Some tests load the content from resources using `getResourceAsStream()`. 
```

Comment:
:+1:
Always a good idea to name a very concrete use case for a user-level requirement.